lib/routes/bulk-get.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+var utils = require('../utils');
+
+module.exports = function (app) {
+  utils.requires(app, 'routes/db');
+
+  // Bulk docs operations
+  app.post('/:db/_bulk_get', utils.jsonParser, function (req, res, next) {
+
+    var opts = req.body || {};
+
+    if (Array.isArray(req.body)) {
+      return utils.sendJSON(res, 400, {
+        error: "bad_request",
+        reason: "Request body must be a JSON object"
+      });
+    }
+    if (!req.body.docs) {
+      return utils.sendJSON(res, 400, {
+        error: "bad_request",
+        reason: "Missing JSON list of 'docs'"
+      });
+    }
+
+    if (typeof req.db.bulkGet !== 'function') {
+      return utils.sendJSON(res, 400, {
+        error: "bad_request",
+        reason: "bulkGet not supported by this version of PouchDB"
+      });
+    }
+
+    // these can be specified via query, e.g. ?revs=true
```

Comment:
Is there a reason why you don't just pass in req.query instead of 'opts' in in the first place? Are there any options the user is not allowed to use? (That's what happens mostly in other places.)